app/docs/[[...slug]]/page.tsx

```
@@ -46,6 +47,56 @@ export default async function Page(props: {
+    ),
+    Accordion,
+    Accordions,
+    Step,
+    Steps,
+    File,
+    Folder,
+    Files,
+    blockquote: Callout as unknown as FC<ComponentProps<'blockquote'>>,
+    APIPage: openapi.APIPage,
+    DocsCategory: ({ slugs = params.slug }: { slugs?: string[] }) => (
+      <DocsCategory page={source.getPage(slugs)!} from={source} />
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)